bftengine/tests/testViewChange/testViewChange.cpp

```diff
@@ -52,28 +52,36 @@ class myViewsManager : public ViewsManager {
 class DummyReplica : public InternalReplicaApi {
```

Comment:
As discussed offline, let's remove this complex infrastructure, and simply replace it with a `MockThresholdVerifier` whose `verify()` method always returns `true`.